python/hail/methods/statgen.py

```diff
@@ -2048,10 +2048,11 @@ def split_multi_hts(ds, keep_star=False, left_aligned=False):
         AD=hl.or_missing(hl.is_defined(ds.AD),
                          [hl.sum(ds.AD) - ds.AD[sm.a_index()], ds.AD[sm.a_index()]]),
         DP=ds.DP,
-        PL=pl
+        PL=pl,
+        GQ=hl.gq_from_pl(pl)
```

Comment:
put GQ before PL here?

Reply:
This is an insert so the overall effect is equivalent, but I've switched it for consistency.